src/content/docs/pt-br/pages/store-jornada/sql/edge-sql-shell-commands.mdx

```diff
@@ -0,0 +1,277 @@
+
+Você receberá uma resposta semelhante a esta:
+
+```sh
+.import: Import data from file files, Kaggle datasets, or databases into a database table.
+
+    Args:
+        arg (str): The import command and its arguments.
+
+    Command Formats:
+        .import file <csv|xlsx> <file_path> <table_name>: Import data from a file CSV or Excel file.
+        .import kaggle <dataset> <data_name> <table_name>: Import data from a Kaggle dataset.
+        .import mysql <database> <source_table> <table_name>: Import data from a MySQL database table.
+        .import postgres <database> <source_table> <table_name>: Import data from PostgreSQL database table.
+        .import turso <database> <source_table> <table_name>: Import data from Turso database.
+
+    Examples:
+        .import file csv /path/to/file.csv my_table
+        .import kaggle joonasyoon/google-doodles list.csv list
+        .import mysql mydb_name source_table_name my_table
+        .import turso <database> <source_table> <table_name>
+```
+
+### .dbsize
+
+Obtém o tamanho do banco de dados atual em MB.
+
+```sh
+EdgeSQL> .dbsize
+```
+
+### .exit
+
+Sai do EdgeSQL Shell.
+
+```sh
+EdgeSQL> .exit
+```
+
+## Variáveis de ambiente
+
+Ao usar o EdgeSQL Shell para importar dados de fontes externas, você pode enviar as credenciais e tokens como variáveis de ambiente usando o comando `export`. Este comando também é utilizado para enviar seu **personal token** da Azion.
+
+### Personal Token
+
+Você pode enviar seu **personal token** como uma variável de ambiente no EdgeSQL Shell.
+
+```sh
+ export AZION_TOKEN="<your_auth_token_here>"
+```
+
+### Kaggle credentials
+
+Use os seguintes comandos para definir suas credenciais do Kaggle:
+
+```sh
+  export KAGGLE_USERNAME="<username>"
+  export KAGGLE_KEY="<kaggle_api_key>"
+```
+
+### Credenciais do MySQL
+
+Use os seguintes comandos para definir suas credenciais do MySQL:
+
+```sh
+  export MYSQL_USERNAME="<username>"
+  export MYSQL_PASSWORD="<password>"
+  export MYSQL_HOST="<host_address>"
+```
+
+Configurações opcionais para MySQL:
+
+```sh
+  export MYSQL_PORT=<port>
+  
+  # For TLS connection
+  export MYSQL_SSL_CA="<ssl_ca>"
+  export MYSQL_SSL_CERT="<ssl_cert>"
+  export MYSQL_SSL_KEY="<ssl_key>"
+  export MYSQL_SSL_VERIFY_CERT=<True|False>
+```
+
+### Credenciais do PostgreSQL
+
+Use os seguintes comandos para definir suas credenciais do PostgreSQL:
+
+```sh
+  export POSTGRES_USERNAME="<username>"
+  export POSTGRES_PASSWORD="<password>"
+  export POSTGRES_HOST="<host_address>"
+```
+
+Configurações opcionais para PostgreSQL:
+
+```sh
+  export POSTGRES_PORT=<port>
+  
+  # For TLS connection
+  export POSTGRES_SSL_CA="<ssl_ca>"
+  export POSTGRES_SSL_CERT="<ssl_cert>"
+  export POSTGRES_SSL_KEY="<ssl_key>"
+  export POSTGRES_SSL_VERIFY_CERT=<True|False>
+```
+
+### Credenciais do Turso
+
+Use os seguintes comandos para definir suas credenciais do Turso:
+
+```sh
+ export TURSO_DATABASE_URL=<https://<db_name>-<organization>.turso.io
```

Comment:
Please, see if this alternative is correct: export TURSO_DATABASE_URL=https://<db_name>-.turso.io>